### PR TITLE
Fix linting

### PR DESCRIPTION
**Pull request by mserranom**
### Description

Fixes linting, which has been broken for a while in `master`.

`build-scripts` are updated to the latest version.

#### Potential Impact

Shouldn't have any, formatting only, and a small change in a couple of loggers.

#### Manual Testing Performed

- [x] Check service works in dev-environment

#### Deployment Checklist

- [ ] Check service continues functioning in staging
- [ ] Check service continues functioning in prod

### CODE REVIEW

File: app/js/ASpellWorker.js [hunk 1]
```diff
@@ -119,10 +122,7 @@ class ASpellWorker {
         this.callback = null
       } else {
         logger.warn(
-          {
```
Comment (jdleesmiller) [inline app/js/ASpellWorker.js:122]: Not one for today... but I think this points at a problem with the linter config. It's restricting to ES2017, and spread is ES2018. I think we could probably bump it up to 2018. CC @timothee-alby @40thieves

Comment (mserranom) [reply app/js/ASpellWorker.js:122]: From https://node.green/#ES2018 it seems node 10 is not 100% compatible with `ES2018`, maybe we need to update to node 12 first (we could even go straight `ES2019` then)

Comment (jdleesmiller) [reply app/js/ASpellWorker.js:122]: Ah, interesting, thanks.

Comment (40thieves) [reply app/js/ASpellWorker.js:122]: Hmm, I'm relatively sure that our version of Node should support spreads like this. I think JLM's initial thought that the linter config is not up-to-date is what I would guess. Possible we need to set the level of support in the config.

Comment (40thieves) [reply app/js/ASpellWorker.js:122]: Indeed, the kangax tables (which is my go-to for JS support) say that Node 8+ should support object spreads: https://kangax.github.io/compat-table/es2016plus/#test-object_rest/spread_properties

Comment (mserranom) [reply app/js/ASpellWorker.js:122]: Yeah it does, has been working here for months. As you say, possibly something to include in the linting config.

**Pull request merged**